src/utils/__tests__/string-test.js

```diff
@@ -1,4 +1,4 @@
-/* @flow strict */
```

Comment:
Did you intend this change?
This is relaxing our type-checking and is undesirable.
Rework the code so this is not needed.

Reply:
This and the below edits were done in order to abstract the Clipboard method for setting the string. `import { Clipboard } from 'react-native'` caused issues with the type-checking, and I could not find a more appropriate file to insert the abstraction into. I don't know what to do otherwise, creating another file just for `Clipboard` seems unwise. Please suggest.